internal/test/wsu/wsu_test.go

```diff
@@ -218,6 +263,28 @@ func runE2ETestSuite(t *testing.T, vm e2ef.WindowsVM, ansibleOutput string) {
 	})
 }
 
+// testDownloadedWMCB checks if the task 'Download WMCB' in the Ansible output was executed, not skipped
+func testDownloadedWMCB(t *testing.T, ansibleOutput string) {
+	// split the output for each task
+	tasks := strings.Split(ansibleOutput, "TASK")
+	// grab the output for the task "Download WMCB"
+	downloadWmcbTaskOutput := ""
+	for i := 0; i < len(tasks); i++ {
+		if strings.Contains(tasks[i], "Download WMCB") {
```

Comment:
Sorry for not catching this earlier but are you sure, you'd always have the string to be same or do you want to ignore the case and see if that matches what we have?

Reply:
It is the name of the Task it is testing, so it should always be present, unless the playbook changes - in which the test should be rewritten anyways
I don't mind changing the case, but I do not see the point of doing so